cmd/kubernetes.go

```diff
@@ -85,7 +85,8 @@ func init() {
 	kubernetesCreateCmd.Flags().BoolVarP(&mergeConfigKubernetes, "merge", "m", false, "merge the config with existing kubeconfig if it already exists.")
 	kubernetesCreateCmd.Flags().BoolVarP(&switchConfigKubernetes, "switch", "", false, "switch context to newly-created cluster")
 	kubernetesCreateCmd.Flags().StringVarP(&existingFirewall, "existing-firewall", "e", "", "optional, ID of existing firewall to use")
-	kubernetesCreateCmd.Flags().StringVarP(&createFirewall, "create-firewall", "c", "", "optional, semicolon-separated list of ports to open - leave blank for default (80;443;6443) or you can use \"all\"")
+	kubernetesCreateCmd.Flags().StringVarP(&rulesFirewall, "firewall-rules", "u", "", "optional, can be used if the --create-firewall flag is set, semicolon-separated list of ports to open")
+	kubernetesCreateCmd.Flags().BoolVarP(&createFirewall, "create-firewall", "c", false, "optional, create a firewall for the cluster with all open ports")
```

Comment:
Is this the desired behaviour? I thought we're opening 80, 443, 6443 by default for Kubernetes firewalls?

Reply:
mm, I will say the user needs to write these ports using the `--firewall-rules` flag